### ISSUE TITLE
android: cannot open file when i download apk by using addAndroidDownloads

### ISSUE THREAD

**Issue by daochaojian** [wkh237/react-native-fetch-blob#449]
when i use DownloadManager to download apk, after download completed i click NOTIFICATION
then i got an error : 
```
DownloadManager:Failed to start Intent { act=Android.intent.action.VIEWdat=content://downloads/all_downloads/9 
typ=application/octet-stream flg=0x3 }:
android.content.ActivityNotFoundException: 
No Activity found to handle Intent { act=android.intent.action.VIEWdat=content://downloads/all_downloads/9 
typ=application/octet-stream flg=0x3 }
``` 

the code i used : 

```
    RNFetchBlob
      .config({
        addAndroidDownloads: {
          useDownloadManager: true,
          notification: true,
          mediaScannable: true,
          mime: 'application/vnd.android.package-archive',
          title: 'app',
          description: 'downloading',
          path: `${RNFetchBlob.fs.dirs.DownloadDir}/app.apk`,
        },
      })
      .fetch('GET', response.data.apkUrl)
      .then(this.downloadApkSuccess)
      .catch(this.downloadApkFail);

  downloadApkSuccess(response) {
    RNFetchBlob.android.actionViewIntent(response.path(), 'application/vnd.android.package-archive');
  }

  downloadApkFail() {
  }
```
it looks like the *mime* did not set successfully,
so i add some codes in *RNFetchBlobReq.java*:

```
            if (options.addAndroidDownloads.getBoolean("useDownloadManager")) {
                Uri uri = Uri.parse(url);
                DownloadManager.Request req = new DownloadManager.Request(uri);
                req.setNotificationVisibility(DownloadManager.Request.VISIBILITY_VISIBLE_NOTIFY_COMPLETED);
                if(options.addAndroidDownloads.hasKey("title")) {
                    req.setTitle(options.addAndroidDownloads.getString("title"));
                }
                if(options.addAndroidDownloads.hasKey("description")) {
                    req.setDescription(options.addAndroidDownloads.getString("description"));
                }
                if(options.addAndroidDownloads.hasKey("path")) {
                    req.setDestinationUri(Uri.parse("file://" + options.addAndroidDownloads.getString("path")));
                }
          // i add this below
                if(options.addAndroidDownloads.hasKey("mime")) {
                    req.setMimeType(options.addAndroidDownloads.getString("mime"));
                }
}
```
i guess it's a bug, so i submit it 
the libraries version i used:
react-native : 0.43.6,
react-native-fetch-blob: 0.10.6.

**Comment by ss7247** [comment]
path: `${RNFetchBlob.fs.dirs.DownloadDir}/app.apk`,   非常重要

**Comment by mohamedtijani** [comment]
@daochaojian you found a solution ?

**Comment by daochaojian** [comment]
@mohamedtijani sorry, i do not  clear which solution  are you looking for, but this issue has been fixed on 0.10.7, hope help you.
